Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN CGO_ENABLED=1 go install -ldflags="-X github.com/segmentio/ctlstore/pkg/vers
   && cp ${GOPATH}/bin/ctlstore-cli /usr/local/bin
 
 FROM alpine
-RUN apk --no-cache add sqlite pigz
+RUN apk --no-cache add sqlite pigz aws-cli perl-utils jq
```

Comment:
not asking for a change here, but I'm curious which thing pulled in perl, was it aws-cli or jq?

Reply:
it was for `shasum` - https://perldoc.perl.org/shasum